### ISSUE TITLE
Feature request: test-mode

### ISSUE THREAD

**Issue by Gabbalo** [circumspect/White-Rabbit#134]
I encountered several times the moment of truth: Is it working, or not?
So I'd suggest a "test-mode" as a new feature, where the whole set of actions is played without 

- the need of players
- the delay of 90 minutes

but within maybe 5 minutes in total.
Everything is done as if there is a whole set of players. Cards are drawn, chats are filled, whatever more.
So you can see as an admin, all cards are shown properly, the channels are working and whatever more :)

**Comment by circumspect** [comment]
Hi,
This already exists! Or, kind of. You still need at least one other discord account to meet the check for number of characters (since you can assign one character role to the bot itself), but you can speed up the game by following the instructions [here](https://white-rabbit.readthedocs.io/en/latest/dev-guide/debugging.html).

**Comment by Gabbalo** [comment]
That'll not work in Docker environment, since you can't access the dev_ids.txt file, there, right? 
File that as a feature request for ENV attributes then maybe ;) 

Also: Discord refuses to grant me another account because they insist of using a cell phone number for registration (I even called for support on them, but no help possible -.- ). And since I only have one sole mobile number, I cannot activate my (existing but inactive) second account - which I would gladly use as a serveradmin for my Alice is Missing Discord to prevent me to see all channel as a player ... thats what I created the second one for in the first place

**Comment by circumspect** [comment]
Makes sense. I'll look into adding a dry run command for admins after I finish the silent falls implementation. Thanks for the feedback!

**Comment by circumspect** [comment]
On looking into the code, it seems like there was already an option to pass a single dev id using the `DEV_ID` environment variable. I've renamed it to `WHITE_RABBIT_DEV_ID` for consistency, however.
